### DISCUSSION TITLE
academy/puppeteer-playwright

### DISCUSSION THREAD

**Discussion by giscus** [apify/apify-docs#574] [Comments]
# academy/puppeteer-playwright

Learn in-depth how to use two of the most popular Node.js libraries for controlling a headless browser - Puppeteer and Playwright.

https://docs.apify.com/academy/puppeteer-playwright

**Comment by JennPlothow** [comment]
# The largest heading
## The second largest heading
###### The smallest heading

**Reply by JennPlothow** [reply]
# The largest heading
## The second largest heading
###### The smallest heading
